Utilities/ReleaseScripts/scripts/MakeBuildSet

```diff
@@ -330,9 +331,8 @@ pitems = []
 if outputFormat != "all":
     outputFormats = [outputFormat]
 for item in outputFormats:
-    pkg = myPackages[item].keys()
-    pkg.sort()
+    pkg = sorted(myPackages[item].keys())
```

Comment:
@smuzaffar for my understanding, is this change really needed or is it just for smarter coding? The ```list.sort()``` should still be available in python3 as far as I can see https://docs.python.org/3/howto/sorting.html

Reply:
I could have used `pkg=list(myPackages[item].keys()); pkg.sort()` but I think current change should be faster if not slower.